src/android/toga_android/window.py

```diff
@@ -41,7 +43,18 @@ def create_toolbar(self):
         pass
 
     def show(self):
-        pass
+        if not self.widget:
+            print("[Android] Showing an empty window is not supported.")
+            return
+
+        # Set the app's entire contentView to this window: only one Window can
+        # be displayed at a time.
+        self.app.native.setContentView(self.widget.native)
```

Comment:
It's a little odd that this is being done in show, rather than `set_content` - it effectively means that if you ever wanted to replace the content of the window, you'd need to re-show().